### PR TITLE
Document for XCache 3.2.0+ auth/unauth origin support

### PR DESCRIPTION
**Pull request by matyasselmeci**
[SOFTWARE-3615: Document XCache 3.2.0 auth/unauth origin variables](https://opensciencegrid.atlassian.net/browse/SOFTWARE-3615)

### CODE REVIEW

**Comment by matyasselmeci** [comment]
(I'll do a squash rebase and bump the 'reviewed date' after this is approved.)

File: docs/data/stashcache/run-stash-origin-container.md [hunk 1]
```diff
@@ -125,8 +158,8 @@ Restart=always
 ExecStartPre=-/usr/bin/docker stop %n
 ExecStartPre=-/usr/bin/docker rm %n
 ExecStartPre=/usr/bin/docker pull opensciencegrid/stash-origin:3.6-release
-ExecStart=/usr/bin/docker run --rm --name %n --publish 1094:1094 \
-  --volume /srv/origin-public:/xcache/namespace \
+ExecStart=/usr/bin/docker run --rm --name %n --publish 1094:1094 --publish 1095:1095 \
```
Comment (edquist) [inline docs/data/stashcache/run-stash-origin-container.md:161]:
```suggestion
ExecStart=/usr/bin/docker run --rm --name %n \
  --publish 1094:1094 \
  --publish 1095:1095 \
```

**Review by edquist — APPROVED** [review]
one possible suggestion, otherwise lgtm

File: docs/data/stashcache/run-stash-origin-container.md (hunk 1 above)
Comment (matyasselmeci) [reply docs/data/stashcache/run-stash-origin-container.md:161]: oh fine :)

**Pull request merged**